procurement_purchase_no_grouping/models/res_config_settings.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 - Radovan Skolnik <radovan@skolnik.info>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    """Configuration of default value for procurement purchase grouping."""
+
+    _inherit = "res.config.settings"
+    # _name = "procurement.purchase.grouping.settings"
+    _description = "Procurement purchase grouping settings"
+
+    procured_purchase_grouping = fields.Selection(
+        [
+            ("standard", "Standard grouping"),
+            ("line", "No line grouping"),
+            ("order", "No order grouping"),
+        ],
+        string="Procured purchase grouping",
+        default="standard",
+        required=True,
+        help="Select the behaviour for grouping procured purchases for the "
+        "the products of this category:\n"
+        "* Standard grouping: Procurements will generate "
+        "purchase orders as always, grouping lines and orders when "
+        "possible.\n"
+        "* No line grouping: If there are any open purchase order for "
+        "the same supplier, it will be reused, but lines won't be "
+        "merged.\n"
+        "* No order grouping: This option will prevent any kind of "
+        "grouping.",
+    )
+
+    @api.model
+    def get_values(self):
+        res = super(ResConfigSettings, self).get_values()
```

Comment:
no need Class name and self if it's the same than current class in python3

Reply:
Good point for a Python newbie ;-) Thanx. Will do.